package.json

```diff
@@ -1,70 +1,71 @@
 {
```

Comment:
@murediane same with this file. I can't see any changes specific to your PR here.

Reply:
it's because of the package I had to install to validate data that was not in the develop package.jason